src/NodaTime/Duration.cs

```diff
@@ -890,17 +925,12 @@ int IComparable.CompareTo(object? obj)
         /// Returns a <see cref="Duration"/> that represents the given number of days, assuming a 'standard' 24-hour
         /// day.
         /// </summary>
+        /// <remarks>If the value does not represent a precise number of nanoseconds, it is rounded, with half values
```

Comment:
The modified `From...` methods mention rounding in `<remarks>`, but `FromNanoseconds` mentions truncation in `<summary>`. Should this be made consistent?

Reply:
Yes, will definitely do that when I get back to this.

Reply:
Help~~~~